pkg/integrations/v2/app_o11y_receiver/config/config.go

```diff
@@ -0,0 +1,98 @@
+package config
+
+import "time"
+
+const (
+	// DefaultRateLimitingRPS is the default value of Requests Per Second
+	// for ratelimiting
+	DefaultRateLimitingRPS = 100
+	// DefaultRateLimitingBurstiness is the default burstiness factor of the
+	// token bucket algorigthm
+	DefaultRateLimitingBurstiness = 50
+	// DefaultMaxPayloadSize is the max paylad size in bytes
+	DefaultMaxPayloadSize = 5e6
+)
+
+// DefaultConfig holds the default configuration of the receiver
+var DefaultConfig = AppO11yReceiverConfig{
```

Comment:
nit: we don't need to explicitly set zero values in the default config.

Reply:
- [ ] dont explicitly set zero values